index.html

```diff
@@ -291,6 +292,109 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="footer">
+      <section id="contact" class="container contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          action="https://formspree.io/f/mvodnloy"
+          method="post"
+          class="form"
+        >
+          <input
+            name="fullname"
+            type="text"
+            placeholder="John Doe"
+            maxlength="30"
+            aria-required="true"
+            required
+          />
+          <input
+            name="email"
+            type="email"
+            placeholder="email@example.com"
+            aria-required="true"
+            required
+            pattern="^[a-z0-9]+@(.+)$"
+          />
+          <textarea
+            name="message"
+            placeholder="write your message here"
+            maxlength="500"
+            aria-required="true"
+            required
+          ></textarea>
+          <button class="btn btn-form" type="submit">Get In Touch</button>
+
+          <span class="error-msg"></span>
+        </form>
+      </section>
+
+      <div class="social">
+        <ul class="social_links">
+          <li>
+            <a
+              href="https://twitter.com/topeogunleye21"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="./media/svg/twitter.svg"
+                alt="link to twitter"
+                class="social_icon"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/ogunleye"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="./media/svg/linkedin.svg"
+                alt="link to linkedin"
+                class="social_icon"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://topeogunleye1.medium.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="./media/svg/medium.svg"
+                alt="link to medium"
+                class="social_icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://github.com/topeogunleye"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="./media/svg/github.svg"
+                alt="link to github"
+                class="social_icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener noreferrer">
+              <img
+                src="./media/svg/angel.svg"
+                alt="Vector icon"
+                class="social_icon"
+            /></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
please your Social Media links Should be aligned at the bottom horizontally and centered